vip-application-importer/src/main/java/fr/insalyon/creatis/vip/applicationimporter/client/view/applicationdisplay/VIPLayout.java

```diff
@@ -84,10 +85,21 @@ public VIPLayout(String width, String height) {
 
         tagsCbItem = createTagsSelect();
 
+        //select list to choose the execution type
+        executionTypeItem = new SelectItem();
+        executionTypeItem.setTitle("<br>Select where the application files must be located</b>");
+        executionTypeItem.setType("comboBox");
+        LinkedHashMap<String, String> executionTypeValueMap = new LinkedHashMap<>();
```

Comment:
replace with `fileAccessProtocolValieMap`

Reply:
Still this on and it is ok.

Reply:
you can check the new changes